### PR TITLE
[styles] More neutral residential garden colours

### PR DESCRIPTION
**Pull request by RedAuburn**
fixes https://github.com/organicmaps/organicmaps/issues/7636

before/after

render
(outdoor style looks the same as the after)

dark before/after

Untitled

vehicle style the same as before

### CODE REVIEW

**Comment by Misalf-git** [comment]
IMO 👍

**Comment by pastk** [comment]
Actually this PR makes the color of regular gardens much lighter also (prev. they used a forest color), which is logical IMO as gardens are often about grass and garden/flower beds, trees are less common than a typical forest.

**Comment by pastk** [comment]
But might be good to test on some bigger complex garden areas like around some palaces.

File: data/styles/clear/include/Basemap.mapcss [hunk 1]
```diff
@@ -136,19 +136,17 @@ area|z14-[landuse=village_green],
 area|z14-[landuse=field],
 {fill-color: @green1;}
 area|z11-12[landuse=forest],
-area|z12[leisure=garden],
-{fill-color: @green2;}
+area|z12-[leisure=garden],
+{fill-color: @green5;}
```
Comment (pastk) [inline data/styles/clear/include/Basemap.mapcss:140]: its good that gardens will have colors distinct from the forests now!

but before z15 garden will be darker than forests which is not logical IMO

and on z15 it'll be the same green5 for both

and from z16- a forest will be darker than a garden (which is fine IMO)

**Review by pastk — CHANGES_REQUESTED** [review]
ATM colors of various greenery become progressively darker when zooming in.

Fixing color of gardens to just one (quite dark) color might create unwanted/confusing combinations on some zoom levels.

IMO gardens should be a bit lighter than forests, but darker than parks.

**Comment by pastk** [comment]
Or maybe separate change of regular non-residential gardens color into a different PR..

**Comment by RedAuburn** [comment]
> Or maybe separate change of regular non-residential gardens color into a different PR..

have removed those changes, i didn't actually mean to do that 😅
should be good now

**Comment by pastk** [comment]
Maybe makes sense to make it more neutral in vehicle style also?
image

**Review by pastk — APPROVED** [review]

**Comment by biodranik** [comment]
The original master version is better than others ) But I won't block the PR, merge if you think it's an improvement.

**Comment by RedAuburn** [comment]
i do think the colour is too saturated at the moment, but this PR isn't a good fix. ideally it'd be almost like the current display at high zoom, with proper fading through the lower zooms. the main issue is that gardens are still highly visible when they stop being rendered because they're too small.

**Pull request closed without merging**